### ISSUE TITLE
Feature request: Support JPEG XL (.jxl) for exports

### ISSUE THREAD

**Issue by C0rn3j** [openscad/openscad#4388]
JPEG XL is a new-generation image format that I would love to see OpenSCAD support for exports.
https://jpegxl.info/

There is a massive difference in size even when losslessly converting a PNG generated by OpenSCAD to JXL:
![image](https://user-images.githubusercontent.com/1641362/196779601-a637e9f8-24e9-4231-9487-ff4be9ee0423.png)

**Comment by t-paul** [comment]
At this point this might only be for Linux, all other don't seem to have the library readily available in their repositories (macOS Homebrew, MXE for Windows). For our macOS builds, this would need someone to help with setting up a universal build.

**Comment by t-paul** [comment]
With Google (as owner of the CLA) [dropping support](https://www.theregister.com/2022/10/31/jpeg_xl_axed_chrome/) due to *"...not enough interest from the entire ecosystem to continue experimenting with JPEG XL"*, I'm closing this for now. In case of growing support, and hopefully better availability in distributions, we can always revisit the topic.

**Comment by C0rn3j** [comment]
I do not believe Alphabet axing support in Chromium with very dubious 'there is no interest' claim should have any bearing on other projects.

https://bugs.chromium.org/p/chromium/issues/detail?id=1178058#c84

There is 70 people in the last 24 hours alone complaining about this decision on that bug thread, clearly there is a lot of interest and I believe I have demonstrated a valid use case.

**Comment by t-paul** [comment]
Mozilla will not support JPEG XL at this time either https://github.com/mozilla/standards-positions/issues/522#issuecomment-1409539985

**Comment by C0rn3j** [comment]
Apple will support JPEG XL. [1]

JPEG XL support keeps being added and improved in other software too. [2]

I ask that you reconsider and reopen the request.

[1] https://www.theregister.com/2023/06/07/apple_safari_jpeg_xl/

[2] https://www.phoronix.com/news/FFmpeg-Animated-JPEG-XL
